.github/workflows/dotnet.yml

```diff
@@ -0,0 +1,28 @@
+name: .NET
```

Comment:
😕 , the name of the file is not great, it should something like "pr-integration"